### PR TITLE
mps: new script for listing MVS processes

### PR DESCRIPTION
**Pull request by MikeFultonDev**

### CODE REVIEW

**Review by tobykeegan — APPROVED** [review]
Tested locally with ZOAU 1.1.0 and 1.2.0, looks good to me!

File: samples/mps.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+		if (NF == 7) {
+			line = substr($0,1,18) "~~~~~~~" substr($0,26)
+		} else {
+			line = $0
+		}
+		if (NF > 6 && text != "") { 
+			print text; text="" 
+		} 
+		{ 
+			text=text line " "
```
[the remark this answers is not in the document]
Comment (MikeFultonDev) [reply samples/mps.sh:23]: good point. i think superfluous. i'll confirm

File: samples/mps.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	BEGIN { text = "" } 
+	{ 
+		sub(/^[ ]+/,""); 
+		if (NF == 7) {
+			line = substr($0,1,18) "~~~~~~~" substr($0,26)
+		} else {
+			line = $0
+		}
+		if (NF > 6 && text != "") { 
+			print text; text=""
```
[the remark this answers is not in the document]
Comment (MikeFultonDev) [reply samples/mps.sh]: done

**Review by AnthonyGiorgio — APPROVED** [review]

**Comment by AnthonyGiorgio** [comment]
Looks good!

**Pull request merged**